styles.css

```diff
@@ -101,12 +119,15 @@ body {
 }
 
 .icon-search {
-  height: 100%;
-  width: 15%;
+  height: 20px;
```

Comment:
I wonder if the icon isn't matching with the desktop because pixels are more of a fixed unit. I think we may want to refactor with a new type of unit.